src/config.rs

```diff
@@ -24,15 +24,16 @@ pub struct CliArgs {
     pub window_attributes: WindowAttributes,
     /// Port number to start a server to listen to remote Firefox devtools connections. 0 for random port.
     pub devtools_port: Option<u16>,
+    /// Path to resource directory. If None, Verso will try to get default directory. And if that
+    /// still doesn't exist, all resource configuration will set to default values.
+    pub resource_dir: Option<PathBuf>,
 }
 
 /// Configuration of Verso instance.
 #[derive(Clone, Debug)]
 pub struct Config {
     /// Global flag options of Servo.
     pub opts: Opts,
-    /// Path to resources directory.
-    pub resource_dir: PathBuf,
```

Comment:
What about we keep `resource_dir` inside `Config`, initialize it in `Config::new` with `args.resource_dir.unwrap_or(resources_dir_path())`

Reply:
sure we can do that.